docs/leaf/key-concepts/leaf-mtg.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Pando Leaf MTG
+sidebar_position: 4
+date: 2021-08-15 22:33:07
+---
+
+Pando Leaf is built with [Mixin Trusted Group](https://developers.mixin.one/document/mainnet/mtg). Right now, there are 6 genesis nodes who serve the trades, they are:
+
+- [Mixin Team](https://mixin.one/): The core develop team of Mixin Network
+- [Fox.ONE](https://fox.one/): The main develop team of 4swap
+- [Poolin](https://poolin.com/): The industry-leading mining service provider
+- [Big.ONE](https://big.one/): A trading platform provides users with the secure and trusteeship services
```

Comment:
--> "A global trading platform that provides users with secure and convenient trusteeship services